### DISCUSSION TITLE
和bilibili直播间工具冲突

### DISCUSSION THREAD

**Discussion by 1686565568** [the1812/Bilibili-Evolved#1323] [综合讨论 / General]
和bilibili直播间工具冲突,
![image](https://user-images.githubusercontent.com/44337928/102326757-c4eb7200-3fbf-11eb-8f6d-efba742d030c.png)不知道是不是和这个有关

**Comment by 1686565568** [comment]
好像是b站刚改版新界面了...

**Comment by the1812** [comment]
直播间弹幕那里改了
